src/bindings/python/tests_compatibility/test_inference_engine/test_ExecutableNetwork.py

```diff
@@ -131,7 +131,8 @@ def test_wait_before_start(device):
   requests = exec_net.requests
   for id in range(num_requests):
       status = requests[id].wait()
-      assert status == ie.StatusCode.INFER_NOT_STARTED
+      # Plugin API 2.0 has the different behavior will not return this status
+      # assert status == ie.StatusCode.INFER_NOT_STARTED
       request_handler = exec_net.start_async(request_id=id, inputs={'parameter': img})
       status = requests[id].wait()
       assert status == ie.StatusCode.OK
```

Comment:
This change won't be a part of this PR. It is ported from CPU plugin branch.

But this change is not compatible with legacy behavior, because behavior in API 2.0 was changed.